image/tools/lib/component/postgres.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+function component_dump_data {
+  dest=$1
+  echo "*:5432:*:${POSTGRES_USERNAME}:${POSTGRES_PASSWORD}" > ~/.pgpass
+  chmod 0600 ~/.pgpass
+  ts=$(date '+%H:%M:%S')
+  export PGPASSFILE=~/.pgpass
+
+  if [ $POSTGRES_SUPERUSER == "true" ]; then
```

Comment:
Getting a bash error here as it needs to use double "[]": `if [[ $POSTGRES_SUPERUSER == "true" ]]; then`

Reply:
I just tried this and it breaks the if statement, so I removed it again!

Reply:
it didn't work for me when running the pod :(

Reply:
I got this error when I ran the job: `bash ==: unary operator expected`

Reply:
built with --no-cache and it worked now